src/DynamoCoreWpf/Properties/Resources.resx

```diff
@@ -1259,12 +1275,54 @@ Don't worry, you'll have the option to save your work.</value>
     <value>New</value>
     <comment>Displays next to the package name in the Package Search window if the package has been added in the last 30 days.</comment>
   </data>
-  <data name="PackageManagerSearchPackagesControlName" xml:space="preserve">
-    <value>packageManagerSearchPackages</value>
-  </data>
   <data name="PackageManagerMyPackagesControlName" xml:space="preserve">
     <value>packageManagerMyPackages</value>
   </data>
+  <data name="PackageManagerSearchTab" xml:space="preserve">
+    <value>Search for Packages</value>
+  </data>
+  <data name="PackageManagerPublishTab" xml:space="preserve">
+    <value>Publish a Package</value>
+  </data>
+  <data name="PackageManagerInstalledPackagesTab" xml:space="preserve">
+    <value>Installed Packages</value>
```

Comment:
Some of these values are already in the resource list, in general I would avoid creating duplicate ones but I want to reach out to localization team first to confirm this principle

Reply:
Confirmed with the localization team this is not an issue for now

Reply:
Right, I didn't think to check if the same title was already present in the resources. Should I leave as is, or reuse the existing one? 
I will check from now on before creating a resource.

Reply:
Let's leave it this way for now